frontend/src/app/main/ui/ds/notifications/toast.cljs

```diff
@@ -9,37 +9,33 @@
    [app.common.data.macros :as dm]
    [app.main.style :as stl])
   (:require
+   [app.common.data :as d]
    [app.main.ui.ds.foundations.assets.icon :as i]
+   [app.main.ui.ds.notifications.shared.notification-pill :refer [notification-pill*]]
    [rumext.v2 :as mf]))
 
-(def ^:private icons-by-level
-  {"info" i/info
-   "warning" i/msg-neutral
-   "error" i/delete-text
-   "success" i/status-tick})
-
 (def ^:private schema:toast
   [:map
    [:class {:optional true} :string]
-   [:level {:optional true}
-    [:maybe [:enum "info" "warning" "error" "success"]]]
+   [:type  {:optional true} [:maybe [:enum :toast :context]]]
+   [:level {:optional true} [:maybe [:enum :info :warning :error :success]]]
    [:on-close {:optional true} fn?]])
 
 (mf/defc toast*
   {::mf/props :obj
    ::mf/schema schema:toast}
-  [{:keys [class level children on-close] :rest props}]
-  (let [class (dm/str (stl/css-case :toast true
-                                    :toast-info (= level "info")
-                                    :toast-warning (= level "warning")
-                                    :toast-error (= level "error")
-                                    :toast-success (= level "success")) " " class)
-        icon-id (or (get icons-by-level level) i/msg-neutral)
-        props (mf/spread-props props {:class class})]
-    [:> "aside" props
-     [:*
-      [:> i/icon* {:icon-id icon-id :class (stl/css :icon)}]
-      children
+  [{:keys [class level type children on-close] :rest props}]
+  (let [class (dm/str class " " (stl/css-case :toast true))
+        level (if (string? level)
+                (keyword level)
+                (d/nilv level :info))
+        type (or type :context)
+        props (mf/spread-props props {:class class :role "alert"})]
+    [:> "div" props
+     [:> notification-pill* {:level level :type type} children]
       ;; TODO: this should be a buttom from the DS, but this variant is not designed yet.
       ;; https://tree.taiga.io/project/penpot/task/8492
-      [:> "button" {:on-click on-close :aria-label "Close" :class (stl/css :close-button)} [:> i/icon* {:icon-id i/close}]]]]))
+     [:> "button" {:on-click on-close
+                   :aria-label "Close"
+                   :class (stl/css :close-button)}
```

Comment:
Not for this PR, but we should check with Design if/when they will be normalizing these kinds of buttons